### ISSUE TITLE
Hack week planning

### ISSUE THREAD

**Issue by fchollet** [keras-team/keras-core#83]
Participants:

- @haifeng-jin 
- @nkovela1 
- @grasskin 
- @hertschuh 
- @mattdangerw 
- @rchao 
- @jbischof 
- @divyashreepathihalli 
- @ianstenbit 
- @qlzh727 
- @sampathweb 
- @chenmoneygithub 

See proposed assignments below.

Some of the assignment buckets may not last the week -- we will reshuffle assignments as we make progress. If we reach a sufficient degree of feature completeness mid-week, we can move on to backwards compatibility testing.

## @fchollet 

- [x] plot_model / model_to_dot
- [x] explore modernization of plot_model
- [x] stack trace filtering
- [x] add support for legacy h5 weights loading
- [x] JAX rnn function
- [ ] FeatureSpace

Layers:
- [x] Wrapper
- [x] Normalization
- [x] Rescaling
- [x] CenterCrop
- [x] Resizing
- [x] RNN / RNNCell
- [x] StackedRNNCells
- [x] SimpleRNN
- [x] LSTM
- [x] GRU
- [x] Lambda
- [ ] ConvLSTM1D/2D/3D
- [x] Bidirectional
- [ ] TimeDistributed

Wrapped layers:
- [x] Discretization
- [x] TextVectorization
- [x] Hashing
- [x] StringLookup
- [x] IntegerLookup

## @chenmoneygithub

- [x] Conv1D/2D/3D
- [x] SeparableConv1D/2D
- [x] DepthwiseConv1D/2D
- [x] Conv1DTranspose/2D/3D

## @sampathweb

- Set up code coverage CI test (e.g. codecov)
- Identify missing tests and major TODOs

Callbacks:
- [x] RemoteMonitor
- [x] TerminateOnNaN
- [x] CSVLogger
- [x] ModelCheckpoint
- [ ] BackupAndRestore

## @qlzh727

- [x] keras-team/keras-core#96
- [x] keras-team/keras#18473

## @ianstenbit

- [x] Confusion metrics
- [x] Learning rate schedules (`keras_core/optimizers/schedules/`)

Callbacks:
- [x] LearningRateScheduler
- [x] ReduceLROnPlateau

## @divyashreepathihalli 

Layers:
- [x] Elu
- [x] ReLU
- [x] LeakyReLU
- [x] PReLU
- [x] Softmax

Wrapped layers:
- [x] RandomCrop
- [x] RandomFlip
- [x] RandomTranslation
- [x] RandomRotation
- [x] RandomZoom

## @jbischof

- Implement `keras_core/backend/pytorch/numpy.py`

## @rchao 

Layers:
- [x] UpSampling1D/2D/3D

## @mattdangerw 

Layers:
- [x] Attention
- [x] AdditiveAttention
- [x] MultiHeadAttention

## @hertschuh 

Layers:
- [x] Flatten
- [x] Reshape
- [x] RepeatVector
- [x] Permute
- [x] Cropping1D/2D/3D
- [x] ZeroPadding1D/2D/3D

## @grasskin 

Metrics
- [x] R2Score
- [x] F1Score
- [x] FBetaScore
- [x] IoU metrics

Callbacks:
- [ ] TensorBoard

## @nkovela1 

Layers:
- [x] UnitNormalization
- [x] LayerNormalization
- [x] GroupNormalization
- [x] SpectralNormalization
- [x] CategoryEncoding
- [x] RandomContrast
- [x] RandomBrightness

## @haifeng-jin 

- [x] Implement step fusing in TF Trainer and JAX Trainer
